pkg/lib/config/config.go

```diff
@@ -45,7 +45,8 @@ var _ = Schema.Add("AppConfig", `
 		"web3": { "$ref": "#/$defs/Web3Config" },
 		"google_tag_manager": { "$ref": "#/$defs/GoogleTagManagerConfig" },
 		"account_migration": { "$ref": "#/$defs/AccountMigrationConfig" },
-		"captcha": { "$ref": "#/$defs/CaptchaConfig" }
+		"captcha": { "$ref": "#/$defs/CaptchaConfig" },
+		"workflow": { "$ref": "#/$defs/WorkflowConfig" }
```

Comment:
I am not sure should we have another file for workflow config instead of mixing it with existing configs.

Reply:
I have put changes in pkg/lib/config/workflow.go. Or do you prefer something else?

Reply:
I meant the config file. Currently all configs are in authgear.yaml. However maybe workflow configs are complicated enough to have its own file?

Reply:
We have three already. I prefer not adding one more.